src/main/java/core/basesyntax/Color.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax;
+
+public enum Color {
+    yellow,
```

Comment:
```suggestion
    YELLOW,
```
please follow enum naming convention